common/spec/lib/factory_bot_helpers.rb

```diff
@@ -576,7 +576,7 @@ def few_or_none(key)
   end
 
   factory :json_bulk_archival_object_updater_job, class: JSONModel(:bulk_archival_object_updater_job) do
-    source { generate(:alphanumstr) }
+    create_missing_top_containers { false }
```

Comment:
@thimios is this related to fixing the api docs generation?

Reply:
@brianzelip this factory was buggy (there is no source property in the bulk archival object update job). The factory is not used in any spec though so we did not realize that it is buggy until I tried to generate the api docs, where all these factories are actually called:
https://github.com/archivesspace/archivesspace/blob/master/docs/api_doc.rb#L116